app/src/main/res/layout-land/question_player_fragment.xml

```diff
@@ -59,6 +59,7 @@
         android:divider="@android:color/transparent"
         android:dividerHeight="8dp"
         android:overScrollMode="never"
+        android:paddingBottom="@dimen/general_screen_padding_bottom"
```

Comment:
Why use a general padding here?

Reply:
@BenHenning I have updated the description. 
To be clear, after adding bottom-padding which was anyway needed as per mocks:
<img width="1534" src="https://user-images.githubusercontent.com/9396084/123153931-e142fc80-d483-11eb-9009-64646860c003.png">


But if we run these two failing test cases on Pixel 3XL (basically a large mobile) all tests pass:
<img width="1534" alt="Screenshot 2021-06-24 at 12 28 54 AM" src="https://user-images.githubusercontent.com/9396084/123153931-e142fc80-d483-11eb-9009-64646860c003.png">